lib/note-info/index.tsx

```diff
@@ -37,43 +30,38 @@ export class NoteInfo extends Component<Props> {
 
   handleClickOutside = this.props.onOutsideClick;
 
-  getNoteLink = (note: T.Note, noteId: T.EntityId) => {
-    const { title } = getNoteTitleAndPreview(note);
-    return `[${title}](simplenote://note/${noteId})`;
-  };
-
-  getPublishURL = (url: string | undefined) => {
-    return isEmpty(url) ? null : `http://simp.ly/p/${url}`;
-  };
-
   render() {
-    const { isMarkdown, isPinned, noteId, note } = this.props;
-    const isPublished = includes(note.systemTags, 'published');
+    const { noteId, note, theme } = this.props;
     const creationDate = note.creationDate * 1000;
     const modificationDate = note.modificationDate
       ? note.modificationDate * 1000
       : null;
-    const publishURL = this.getPublishURL(note.publishURL);
-    const noteLink = this.getNoteLink(note, noteId);
 
     return (
-      <div className="note-info theme-color-bg theme-color-fg theme-color-border">
-        <div className="note-info-panel note-info-stats theme-color-border">
-          <div className="note-info-header">
-            <PanelTitle headingLevel={2}>Info</PanelTitle>
+      <Modal
+        key="note-info-modal"
+        className="dialog-renderer__content note-info theme-color-border theme-color-bg theme-color-fg"
+        contentLabel="Document"
+        isOpen
+        onRequestClose={this.handleClickOutside}
+        overlayClassName="dialog-renderer__overlay"
+        portalClassName={`dialog-renderer__portal theme-${theme}`}
+      >
+        <div className="note-info-panel note-info-stats theme-color-border theme-color-fg-dim">
+          <div className="note-info-header theme-color-border">
+            <h2 className="panel-title theme-color-fg">Document</h2>
             <button
               type="button"
```

Comment:
Since this `button` does not include a text string, we should add a helpful ARIA label to describe the element for screen reader users.
```suggestion
              aria-label="close modal"
              type="button"
```

Reply:
I've added an aria-label here now.